elixir/word-count/word_count.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Words do
+  @pattern ~r/[a-zA-Z0-9-\p{L}]+/u
```

Comment:
I forgot to mention this before you moved the pattern, but now it is easier to comment the correct line:

The `\p{L}` part matches unicode "letters", which already contains the `a-zA-Z0-9` part.
This could be reduced to `-\p{L}`